### ISSUE TITLE
[BITV]: Menu item contrasts are not enough

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#42688]
On: `index.php/u/admin`

Text have to be "white"
![Screenshot from 2024-01-10 17-40-41](https://github.com/nextcloud/server/assets/6078378/43b81cfc-b82b-4fd3-bb67-d2ec54dc2da9)

Text have to be "black"
![Screenshot from 2023-12-22 11-26-08](https://github.com/nextcloud/server/assets/6078378/1c32f5ac-de2f-4331-8c16-9089c523dd1b)
